src/Facility.Definition/Fsd/FsdParsers.cs

```diff
@@ -40,7 +40,7 @@ internal static class FsdParsers
 			parser.Bracketed(PunctuationParser(openBracket), PunctuationParser(closeBracket));
 
 		private static IParser<Match> AttributeParameterValueParser { get; } =
-			Parser.Regex(@"""(([^""\\]+|\\[""\\/bfnrt]|\\u[0-9a-fA-f]{4})*)""|([0-9a-zA-Z.+_-]+)");
+			Parser.Regex(@"""(([^""]+|\\[""\\/bfnrt]|\\u[0-9a-fA-f]{4})*)""|([0-9a-zA-Z.+_-]+)");
```

Comment:
This was the issue with the tests I wrote. I imagine this was part of the regex for a reason, but I'm not sure what that reason is.

Reply:
Thanks tests.

It appears to be related to JSON-valued attribute parameters. I'm not sure I understand their purpose in FSD. I've not needed anything like that in practice and the docs don't mention them AFAICT.

Would it make sense to delete that test if they aren't supported/useful in the language?

Reply:
FSD attribute string values support backslash escapes. If you want a literal backslash in a regex string, you'll have to double-backslash it, e.g. `[validate(regex: "^\\d+$")]`.

Reply:
Ah. I knew `[""\\/bfnrt]` seemed familiar…

That adds more value to the idea of using a different delimiter for regex literals. I find regular expressions to be much easier to understand without having to escape \, since there are a lot of them on average. Would you consider the JS regex syntax?

i.e. `[validate(regex: /^\d+&/)]`

Reply:
I have no objection to supporting regex-style string escaping.

`Parser.Regex(@"""(([^""\\]+|\\[""\\/bfnrt]|\\u[0-9a-fA-f]{4})*)""|/([^/\\]+|\\[^\r\n])*/|([0-9a-zA-Z.+_-]+)");`

Reply:
That regex causes some test failures. It isn't immediately obvious as to why. I tried adding another capture group between the slashes (including the Kleene star). I'll try to look into that next time I sit down to work on this.

Reply:
The reason that regular expression fails is pretty murky. I just went with the usual string literals for the moment, but perhaps someone will be able to figure out the parser in the meantime. I'll try and look at it again once the other pieces are in place.

Reply:
The combinator stuff threw me at first. I missed `TryParseAttributeParameterValue` the first time 'round by assuming the capture group mapping happened in the FaithlifeParsing 🤦‍♂️